src/index.html

```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="page__header">
+      <a href="./index.html" class="logo"><img src="./images/logo.png" alt="logo"></a>
```

Comment:
fomartting seems to be little off, are you using autoformatting? 🤔

Reply:
I've been trying to disable it. :)

Reply:
why exactly? code is more readable when correctly formatted 😄

Reply:
And the core word is 'rorrectly' :)